agents/craftassist/craftassist_agent.py

```diff
@@ -162,6 +163,10 @@ def setup_agent_initial_state(sid):
             }
             sio.emit("setVoxelWorldInitialState", payload)
 
+        @sio.on("get_agent_type")
```

Comment:
This function can be put into loco_mc_agent.py instead of duplicating in craftassist_agent.py & locobot_agent.py

Reply:
updated